### ISSUE TITLE
PaymentRequestButtonElement -- Click does not open Browser Payment Interface

### ISSUE THREAD

**Issue by ColinVDH** [stripe/react-stripe-elements#123]
#### Environment
Chrome 62, Windows 10

#### Steps to Reproduce
1. Ensure that there is a valid card saved in the browser. 
2. Add the "--https" flag to the demo script and run the demo. The button shows up but click does not open the Browser Payment Interface.   

Note that in this same environment, the [button element from the docs](https://stripe.com/docs/stripe-js/elements/payment-request-button#try-it-out) works perfectly

**Comment by atty-stripe** [comment]
Hi @ColinVDH! Thanks for the report, we'll investigate. Do you see any warnings or errors in the browser console?

**Comment by ColinVDH** [comment]
Thanks @atty-stripe. On click, the following warning appears: 

> Do not call show() yourself if you are using the paymentRequestButton Element.
> The Element handles showing the payment sheet.

**Comment by jez** [comment]
Hey @ColinVDH, I looked into this. On a fresh clone + yarn install, I couldn't reproduce the issue using the demo page on Chrome 62 + Windows 10 nor Chrome 63 + macOS.

It's possible that the demo doesn't play nicely using the `--https` flag to `webpack-dev-server`. Here at Stripe, use use ngrok to proxy https requests the http localhost server. This *might* solve your issue, but I can't think of a reason why `--https` wouldn't work. (I tried to get `--https` working, but it wouldn't even show me the payment request button because canMakePayment was false when using that flag).
